R/gen_OpenSpecy.R

```diff
@@ -62,6 +62,15 @@ plot.OpenSpecy <- function(x, ...) {
   ...)
 }
 
+#' @rdname gen_OpenSpecy
+#'
+#' @method lines OpenSpecy
+#' @export
+lines.OpenSpecy <- function(x, ...) {
+  matlines(x$wavenumber, x$spectra, type = "l",
+          ...)
+}
+
 #Could be nice to add at some point.
 #summary.OpenSpecy <- function(x, ...) {
 
```

Comment:
What should the output of this `summary()` function look like? I could quickly code one ..

Reply:
I was thinking a few common things people want, number of spectra, resolution of wavenumbers, number of metadata variables, min and max wavenumbers, number of wavenumbers values, max and min intensity values across all spectra, max and min x and y values from the metadata.